meta-balena-common/recipes-containers/hostapp-extensions-update/hostapp-extensions-update.bb

```diff
@@ -0,0 +1,35 @@
+DESCRIPTION = "BalenaOS hostapp extensions updater"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+	file://update-hostapp-extensions \
+	file://update-hostapp-extensions.service \
```

Comment:
I might be missing something but is there a reason to have a service and run it with `systemd-run` rather than just running it as a standart bash script (similar to `HUP`).

Reply:
The only reason is to have the logs centralized in journald. I expect balenahup to run the service and to be able to see the logs in the dashboard.